packages/components/atoms/f-image-tile/test/accessibility/axe-accessibility.spec.js

```diff
@@ -0,0 +1,18 @@
+import { getAccessibilityTestResults } from '../../../../../../test/utils/axe-helper';
+
+const ImageTile = require('../../test-utils/component-objects/f-imageTile.component');
+const imageTile = new ImageTile();
+
+describe('Accessibility tests', () => {
+    beforeEach(() => {
+        imageTile.open();
+        imageTile.waitForComponent();
```

Comment:
Does this run and pass? ... I only ask because the Generator currently produced Chrome and Accessibility Tests that fail.
```suggestion
        imageTile.load();
```

Reply:
updated!